RocksDB Building error

```cs
Plugins/Storage/RocksDBStore.cs(10,30): error CS0507: 'RocksDBStore.Configure()': cannot change access modifiers when overriding 'protected' inherited member 'Plugin.Configure()' [/opt/neo-plugins/src/RocksDBStore/RocksDBStore.csproj]
```

@shargon, do you have an idea?

--- a/src/RocksDBStore/Plugins/Storage/RocksDBStore.cs
+++ b/src/RocksDBStore/Plugins/Storage/RocksDBStore.cs
@@ -7,7 +7,7 @@ namespace Neo.Plugins.Storage
         /// <summary>
         /// Configure
         /// </summary>
-        public override void Configure()
+        protected override void Configure()
         {
             Settings.Load(GetConfiguration());
         }

--- a/src/RocksDBStore/RocksDBStore.csproj
+++ b/src/RocksDBStore/RocksDBStore.csproj
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00810" />
+    <PackageReference Include="Neo" Version="3.0.0-CI00846" />
     <PackageReference Include="RocksDbNative" Version="6.2.2" />
     <PackageReference Include="RocksDbSharp" Version="6.2.2" />
   </ItemGroup>

